pkg/share/manager/json/json.go

```diff
@@ -393,7 +398,29 @@ func (m *mgr) ListReceivedShares(ctx context.Context, filters []*collaboration.F
 			rss = append(rss, rs)
 		}
 	}
-	return rss, nil
+
+	// if there is a mix-up of shares of type group and shares of type user we need to deduplicate them, since it points
+	// to the same resource. Leave the more explicit and hide the more explicit. In this case we hide the group shares
```

Comment:
this should be: 

> Leave the more explicit and hide the more implicit

Reply:
Used `less explicit` instead `more implicit`. I think that is clearer. See https://github.com/cs3org/reva/pull/2383